setup.py

```diff
@@ -33,14 +44,9 @@
 for reqs in extras_require.values():
     extras_require['all'].extend(reqs)
 
-setup_requires = [
-    'pytest-runner>=2.6.2',
-]
-
 install_requires = [
-    'Flask>=0.10',
-    'celery>=3.1;python_version<"3.7"',
-    'celery>=4.3;python_version>="3.7"',
+    'Flask>=1.1',
+    'celery>=5.1.0',
```

Comment:
Comment: I've just checked and it indeed seems like only 5.1+ is supported now (https://github.com/celery/celery/blob/master/SECURITY.md) despite 4.x being declared an LTS release and 6.x haven't been released yet.